storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManager.java

```
@@ -973,6 +973,9 @@ public void copyLogSegmentsToRemote(UnifiedLog log) throws InterruptedException
                             segmentIdsBeingCopied.add(segmentId);
                             try {
                                 copyLogSegment(log, candidateLogSegment.logSegment, segmentId, candidateLogSegment.nextSegmentOffset);
+                            } catch (RemoteStorageException e) {
```

can we catch all exception in the `catch` block and then throw it?

RemoteStorageException -> Exception

since `copyLogSegment` throws InterruptedException, ExecutionException, RemoteStorageException, IOException, and CustomMetadataSizeLimitExceededException:

```
catch (Exception e) {
  recordLagStats(log);
  throw e;
}  
```